app/controllers/api/articles_controller.rb

```diff
@@ -22,6 +23,8 @@ def show
   def create
     authorize Article.create
     article = Article.create(article_params)
+    article.category = Category.find_by name: params["article"]["category"]
```

Comment:
Format your code

Reply:
article.category = Category.find_by name: params['article']['category']
this is the end result.
Were the double quotes the issue?